controllers/addresspool_controller.go

```diff
@@ -43,7 +43,10 @@ type AddressPoolReconciler struct {
 	Namespace string
 }
 
-const RetryPeriod = 5 * time.Minute
+const (
+	RetryPeriod          = 5 * time.Minute
+	MetalLBConfigMapName = "config"
+)
 
 var AddressPoolManifestPath = "./bindata/configuration/address-pool"
```

Comment:
nit: this can also be a const

Reply:
no this var get modified in the unit test hence we need to have it as variable